src/plugins/activityStates/plugin.js

```diff
@@ -0,0 +1,42 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2023, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import activityStatesInterceptor from './activityStatesInterceptor.js';
+import { createActivityStatesIdentifier } from './createActivityStatesIdentifier.js';
+
+const ACTIVITYSTATES_DEFAULT_NAME = 'Activity States';
+
+export default function ActivityStatesPlugin(
+  name = ACTIVITYSTATES_DEFAULT_NAME,
+  namespace = '',
+  priority = undefined
```

Comment:
Since it has 3+ parameters, our style guide says we should wrap these in an `options` object. We should also define it using JSDocs.